google_drive/google_calendar.py

```diff
@@ -35,62 +28,92 @@ class GoogleCalendarBlock(XBlock):
         display_name="Display Name",
         help="This name appears in the horizontal navigation at the top of the page.",
         scope=Scope.settings,
-        default="Example Google Calendar"
+        default="Google Calendar"
     )
 
-    embed_code = String(
-        display_name="Embed Code",
-        help="Google provides an embed code for Drive calendars with a variety of settings. From inside a Google Drive calendar, select Publish to the Web from within the File menu to get to your Embed Code with your desired settings.",
+    calendar_id = String(
+        display_name="Public Calendar ID",
+        help=(
+            "Google provides an ID for publicly available calendars. In the Google Calendar, "
+            "open Settings and copy the ID from the Calendar Address section into this field."
+        ),
         scope=Scope.settings,
-        default=textwrap.dedent("""
-            <iframe
-                src="https://docs.google.com/presentation/d/1B22AsMwG7jgE39vm1BuiJClir_JUz1q077cPEKJR2mI/embed?start=false&loop=false&delayms=3000"
-                frameborder="0"
-                width="960"
-                height="569"
-                allowfullscreen="true"
-                mozallowfullscreen="true"
-                webkitallowfullscreen="true">
-            </iframe>
-        """))
-
-    def student_view(self, context={}):
+        default="edx.org_lom804qe3ttspplj1bgeu1l3ak@group.calendar.google.com"
```

Comment:
Is this a generic google account that we use for bootstrapping the component?  Should we move this value into a configuration setting?